recipe/meta.yaml

```diff
@@ -46,7 +46,6 @@ requirements:
 
 test:
   commands:
-    - votca_compare --help
```

Comment:
@junghans Can you tell me which tools were removed in 2023? that would speed up the process to build the updated version and once it is working I can push the update for 2024.

Reply:
Only `votca_compare` and `votca_help2doc`.